courses/machine_learning/deepdive2/structured/solutions/4a_sample_babyweight.ipynb

```diff
@@ -0,0 +1,2818 @@
+{
```

Comment:
<p>Similarly, this whole cell could be replaced by</p><p> </p><p>display_dataframe_head_from_query(data_query)</p><p> </p><p>By breaking up query_string</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Done</p>